ShortCutCoreSegmenter::RefineShortCut: Fix windows link error

```
Error	6	error LNK2019: unresolved external symbol "void __cdecl 
TCGA::FindBoundingBoxFromMask(class cv::Mat const &,class cv::Rect_<int> &)" 
(?FindBoundingBoxFromMask@TCGA@@YAXAEBVMat@cv@@AEAV?$Rect_@H@3@@Z) 
referenced in function "public: void __cdecl ShortCutCoreSegmenter::RefineShortCut(void)" 
(?RefineShortCut@ShortCutCoreSegmenter@@QEAAXXZ)	C:\D\N\S-1-E-b\SlicerPathology-
build\ShortCutCore\Logic\ShortCutCoreSegmenter.obj	vtkSlicerShortCutCoreModuleLogic

```

--- a/QuickTCGA/Logic/TCGAUtilities.h
+++ b/QuickTCGA/Logic/TCGAUtilities.h
@@ -23,6 +23,8 @@
 
 #include <csignal>
 
+#include "vtkSlicerQuickTCGAModuleLogicExport.h"
+
 namespace TCGA
 {
 
@@ -57,11 +59,14 @@ void CopyImageOpenCV2VTK(const cv::Mat &imOpenCV, vtkSmartPointer<vtkImageData> 
 
 
 
-void CopyImageVTK2OpenCV(const vtkSmartPointer<vtkImageData> imVTK, cv::Mat &imOpenCV);
+void VTK_SLICER_QUICKTCGA_MODULE_LOGIC_EXPORT
+CopyImageVTK2OpenCV(const vtkSmartPointer<vtkImageData> imVTK, cv::Mat &imOpenCV);
 
-void CopyImageOpenCV2VTK(const cv::Mat &imOpenCV, vtkSmartPointer<vtkImageData> imVTK);
+void VTK_SLICER_QUICKTCGA_MODULE_LOGIC_EXPORT
+CopyImageOpenCV2VTK(const cv::Mat &imOpenCV, vtkSmartPointer<vtkImageData> imVTK);
 
-void FindBoundingBoxFromMask(const cv::Mat& imMask, cv::Rect& rect);
+void VTK_SLICER_QUICKTCGA_MODULE_LOGIC_EXPORT
+FindBoundingBoxFromMask(const cv::Mat& imMask, cv::Rect& rect);
 
 
 

